src/components/DepositWidget/Row.tsx

```diff
@@ -75,6 +163,20 @@ export const Row: React.FC<RowProps> = (props: RowProps) => {
   const [visibleForm, showForm] = useState<'deposit' | 'withdraw' | void>()
   const exchangeBalanceTotal = exchangeBalance.add(depositingBalance)
 
+  // Checks innerWidth
+  let showResponsive = innerWidth < 500
+
+  useMemo((): void => {
+    const bodyClassList: string[] | DOMTokenList = (window && window.document.body.classList) || []
+    const noScrollActive = Array.from(bodyClassList).find((className: string): boolean => className === 'noScroll')
+    // if innerWidth > 500 && body has noScroll active - REMOVE it
+    if (noScrollActive && !visibleForm && !showResponsive) {
+      ;(bodyClassList as DOMTokenList).remove('noScroll')
```

Comment:
How is modifying the `classList` of the body responsibility of a `DepositWidget row` component?

Reply:
accessing `visibleForm` is a bit difficult elsewhere

Reply:
normally I'd do this via a `middleware` but not sure how here with our structure of data and setters being rather local

Reply:
made into a hook